src/main/java/com/hydratereminder/dictionary/HydrateBreakMessageDictionary.java

```diff
@@ -259,6 +260,31 @@ public static String getRandomHydrateBreakMessageForPersonality(HydrateReminderP
             case AGGRESSIVE:
                 breakMessage = getRandomBreakMessage(HYDRATE_BREAK_AGGRESSIVE_TEXT_LIST);
                 break;
+            case RANDOM:
+                int randomNumber = ThreadLocalRandom.current().nextInt(1, 7);
+                switch (randomNumber) {
+                    case 1 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_FUNNY_TEXT_LIST);
+                        break;
+                    case 2 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_CARING_TEXT_LIST);
+                        break;
+                    case 3 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_ROMANTIC_TEXT_LIST);
+                        break;
+                    case 4 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_POLITE_TEXT_LIST);
+                        break;
+                    case 5 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_MOTIVATIONAL_TEXT_LIST);
+                        break;
+                    case 6 :
+                        breakMessage = getRandomBreakMessage(HYDRATE_BREAK_AGGRESSIVE_TEXT_LIST);
+                        break;
```

Comment:
It looks like a few personality types are left out of the random calculation. Can you make sure to include all personality types?

Reply:
I will make a commit on your approval @jmakhack

Reply:
@akshatjainei, this looks good for now. Go ahead and commit your changes so I can review the diff on Github.